src/test/java/com/google/javascript/gents/singleTests/fields.js

```diff
@@ -11,6 +11,11 @@ var A = function(a) {
   let y = 1;
   /** @type {number} */
   this.z = y + 1;
+
+  this.w;
+  this.w.bar = 'bar';
+
+  this.n = 13;
```

Comment:
Actually, never mind. To do this right I need a different approach so I'm going to just do that directly.